connect/src/main/java/com/stripe/android/connect/webview/serialization/AppearanceJs.kt

```diff
@@ -362,15 +362,24 @@ internal data class AppearanceVariablesJs(
     val labelSmTextTransform: TextTransform?,
 )
 
+private fun Float?.toPx(): String? {
+    return this?.let { "${it.toString().removeSuffix(".0")}px" }
+}
```

Comment:
This is a bit cleaner, less string manipulation:
```suggestion
private fun Float?.toPx(): String? {
    return this?.let { "${roundToInt()}px" }
}
```